packages/plugin-char/CharFormatDomRenderer.ts

```diff
@@ -19,7 +19,10 @@ export class CharFormatDomRenderer extends InlineFormatDomRenderer {
         if (attributeNames.length) {
             const span = document.createElement('span');
             for (const name of attributeNames) {
-                span.setAttribute(name, node.attributes[name]);
+                const value = node.attributes[name];
+                if (typeof value === 'string') {
+                    span.setAttribute(name, value);
+                }
```

Comment:
And now *all* renderers are defiled by this `typeof`... :( Hopefully it will go away when merging (back) formats and attributes into a single concept.

Reply:
I know... 
![image](https://user-images.githubusercontent.com/24205914/74421960-24339f80-4e4e-11ea-87bd-4dacb5ac9cd8.png)